ProjectLighthouse.Servers.Website/Controllers/Admin/AdminUserController.cs

```diff
@@ -143,7 +143,22 @@ public async Task<IActionResult> WipeScores([FromRoute] int id)
     }
 
     /// <summary>
-    ///     Forces the email verification of a user.
+    /// Deletes the user's current avatar. Can prevent crashes in-game, or just be used to remove images that break guidelines.
+    /// </summary>
+    [HttpGet("wipeAvatar")]
+    public async Task<IActionResult> WipeAvatar([FromRoute] int id)
+    {
+        UserEntity? user = this.database.UserFromWebRequest(this.request);
+        if (user == null || !user.IsModerator) return this.NotFound();
+
+        UserEntity? targetedUser = await this.database.Users.FirstOrDefaultAsync(u => u.UserId == id);
+        if (targetedUser == null) return this.NotFound();
+
+        targetedUser.IconHash = "";
```

Comment:
```suggestion
        targetedUser.IconHash = "";
        
        await this.database.SaveChangesAsync();
        Logger.Success($"Reset profile picture for {targetedUser.Username} (id:{targetedUser.UserId})", LogArea.Admin);

        await this.database.SendNotification(targetedUser.UserId, "Your profile picture has been reset by a moderator.");
        
        return this.Redirect($"/user/{targetedUser.UserId}");
```
To keep this consistent with other functions in this file like `WipeComments` and `WipeScores`, we should log that the person's picture has been reset and send them a notification.

Also in C# and EntityFramework when you modify a database entity, your changes are not saved until you call a variation of `SaveChanges()`.